lua/telescope/builtin/files.lua

```diff
@@ -148,7 +148,22 @@ end
 -- TODO: Maybe just change this to `find`.
 -- TODO: Support `find` and maybe let people do other stuff with it as well.
 files.find_files = function(opts)
-  local find_command = opts.find_command
+  local find_command = (function()
+    if opts.find_command then
+      return opts.find_command
+    elseif 1 == vim.fn.executable "fd" then
+      return { "fd", "--type", "f" }
+    elseif 1 == vim.fn.executable "fdfind" then
+      return { "fdfind", "--type", "f" }
+    elseif 1 == vim.fn.executable "rg" then
+      return { "rg", "--files" }
+    elseif 1 == vim.fn.executable "find" and vim.fn.has "win32" == 0 then
+      return { "find", ".", "-type", "f" }
+    elseif 1 == vim.fn.executable "where" then
+      return { "where", "/r", ".", "*" }
+    end
+  end)()
+  local command = find_command[1]
```

Comment:
One last thing i just noticed while doing the final review. If we never find a command `find_command` is nil which would cause a error here. 

In line 228 is the check. We need to do that check now earlier before we actually do anything with `find_command`

Reply:
done